src/Arcus.API.Market/Repositories/OrderRepository.cs

```diff
@@ -60,8 +62,20 @@ public async Task OrderBaconAsync(int amount)
                 finally
                 {
                     var serviceBusEndpoint = _serviceBusOrderSender.FullyQualifiedNamespace;
+                    string entityPath = _serviceBusOrderSender.EntityPath;
                     _logger.LogInformation($"Done sending at {DateTimeOffset.UtcNow}");
-                    _logger.LogServiceBusQueueDependency(_serviceBusOrderSender.EntityPath, isSuccessful, serviceBusDependencyMeasurement, dependencyId: newDependencyId);
+                    //_logger.LogServiceBusQueueDependency(serviceBusEndpoint, entityPath, isSuccessful, serviceBusDependencyMeasurement, dependencyId: newDependencyId);
+                    _logger.LogWarning(MessageFormats.DependencyFormat, new DependencyLogEntry(
```

Comment:
Problem with the Service Bus dependency tracking: we should only add the entity path to the target name, not the namespace. Fixed in https://github.com/arcus-azure/arcus.observability/pull/445

Reply:
Should we then first release a new version of observability that contains that fix, before updating this POC ?

Reply:
We could, indeed. 👍

Reply:
We should already releasing v2.6 of observability as we have several new features on `master` beside this fix.

Reply:
Upgraded with Observability v2.6 and WebAPI v1.6.1.

Reply:
Still correct application map and request linking. 👍